src/components/visualizations/chart/highcharts/plugins/zeroAlignYAxis.ts

```diff
@@ -104,7 +106,7 @@ function getRange(targetNumber: number, delta: number): number[] {
     let position = 0;
     const result = [];
 
-    const limit = Math.abs(targetNumber);
+    const limit = +Math.abs(targetNumber).toFixed(FRACTION_DIGITS);
```

Comment:
I see that this "hack" with + is hard to decode. Please add the link to stack overflow question where we found it to the comment of this line

Reply:
Added comment